src/components/ClusterOverview/TopConsumers/TopConsumers.js

```diff
@@ -1,100 +1,188 @@
 import React from 'react';
-import { UtilizationBar } from 'patternfly-react';
-
 import PropTypes from 'prop-types';
+import { get } from 'lodash';
 
 import { getFormElement } from '../../Form';
 import { DROPDOWN } from '../../Form/constants';
-import { PodModel, VirtualMachineModel } from '../../../models';
 import {
   DashboardCard,
   DashboardCardBody,
   DashboardCardHeader,
   DashboardCardTitle,
   DashboardCardTitleHelp,
 } from '../../Dashboard/DashboardCard';
-import { ClusterOverviewContextGenericConsumer } from '../ClusterOverviewContext';
+import { ClusterOverviewContext } from '../ClusterOverviewContext';
+import { InlineLoading } from '../../Loading';
+import { formatBytes as utilFormatBytes } from '../../../utils';
+import { getConsumers } from '../../../selectors';
+import { ConsumerItem } from '../../Dashboard/TopConsumers/ConsumerItem';
+import { ConsumersResults } from '../../Dashboard/TopConsumers/ConsumersResults';
+import {
+  BY_CPU,
+  BY_NETWORK,
+  BY_STORAGE,
+  BY_MEMORY,
+  PODS_AND_VMS,
+  INFRASTRUCTURE,
+  NODES,
+  WORKLOADS,
+  CPU_DESC,
+  MEMORY_DESC,
+  STORAGE_DESC,
+  NETWORK_DESC,
+} from './strings';
+import { ConsumersFilter } from '../../Dashboard/TopConsumers/ConsumersFilter';
 
-const sortConsumers = (metrics, filter, sortBy) => {
-  const metricKey = Object.keys(metrics).find(key => metrics[key].title === sortBy);
-  const filteredConsumers =
-    filter !== 'All' ? metrics[metricKey].consumers.filter(c => c.kind === filter) : metrics[metricKey].consumers;
-  const max = Math.max(...filteredConsumers.map(c => c.usage));
-  return filteredConsumers
+const getMetricConsumers = ({ consumers }) => {
```

Comment:
this should probably go to selectors or some util class (Dashboard/TopConsumers/utils or some other utils?)

Reply:
`getMetricConsumers` moved to Dashboard/TopConsumers/utils. The other two methods were moved to ClusterOverview/TopConsumers/utils